src/main/java/ua/lviv/iot/managers/ConcertHallManager.java

```diff
@@ -0,0 +1,61 @@
+package ua.lviv.iot.managers;
+
+
+import lombok.Data;
+import lombok.NonNull;
+import ua.lviv.iot.models.Performance;
+import ua.lviv.iot.models.SortOrder;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.LinkedList;
+
+@Data
+@NonNull
+public final class ConcertHallManager {
+
+    private LinkedList<Performance> performances;

```

Comment:
Why would you use LinkedList? Do you think there might be a more appropriate class to use here?

Reply:
It is ArrayList, fixed